src/Microsoft.DotNet.ProjectModel/Graph/LockFileReader.cs

```diff
@@ -144,6 +144,9 @@ private void ReadLibrary(JObject json, LockFile lockFile)
 
                 var type = _symbols.GetString(value.Value<string>("type"));
 
+                var pathValue = value["path"];
```

Comment:
See test `ReadLockFileReadsLibrariesWithoutPathProperty` below. Additionally, this code was previously used for project libraries.